tox.ini

```diff
@@ -42,11 +42,22 @@ commands =
 
 [testenv:integrationtests]
 basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY BRANCH INTEGRATION_TESTS_REPO_URL
 deps =
 	-rrequirements/tests.txt
+setenv =
+	INT_TESTS_REPO_SSH_URL = git@github.com:gridsingularity/gsy-backend-integration-tests.git
+whitelist_externals =
+	git
+	bash
+commands_pre =
```

Comment:
Are you sure that this is working? If I'm not mistaken, this has been introduced in `tox==3.4` and we're using `tox==3.24`. Could you please double-check?